services/backend-for-frontend-nodejs/src/index.ts

```diff
@@ -1,20 +1,22 @@
-import "./tracing"
+// import { trace, context, ROOT_CONTEXT, SpanStatusCode } from '@opentelemetry/api'; // Import modules from OTel
+// Step 1 Add Span Attributes and Create Custom Spans. Import trace module
```

Comment:
Since we are referencing steps should we also reference which acitivity we are referring to?